packages/luna/src/components/TutorFeedback/RatingDialog.tsx

```diff
@@ -70,9 +77,12 @@ export const EditRating: React.FC<FeedbackEditProps> = ({
         <Button
           size={ButtonSize.Large}
           onClick={() =>
-            onUpdate({ value: newRating, feedback: newFeedback || null })
+            onSubmit({ value: newRating, feedback: newFeedback || null })
           }
-          disabled={newFeedback === feedback && newRating === rating}
+          disabled={
+            (newFeedback === feedback && newRating === rating) || loading
```

Comment:
Just a simple note: consider using the short simple expression first in the (or) condition.